flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/io/benchmark/StreamNetworkBenchmarkEnvironment.java

```diff
@@ -158,6 +157,10 @@ public void tearDown() {
 		suppressExceptions(receiverEnv::close);
 	}
 
+	/**
+	 * Note: It should be guaranteed that {@link #createResultPartitionWriter(int)} has been called before
+	 * creating the receiver.
```

Comment:
nit: might supplement the reason why we should guarantee this. 
E.g. `Otherwise it might cause unexpected behaviors when {@link PartitionNotFoundException} happens in {@link TestRemoteInputChannel}`.